cmd/deploy.go

```diff
@@ -13,24 +14,44 @@ type DeployCmd struct {
 	ui              boshui.UI
 	deployment      boshdir.Deployment
 	releaseUploader ReleaseUploader
+	director        boshdir.Director
 }
 
 type ReleaseUploader interface {
 	UploadReleases([]byte) ([]byte, error)
 	UploadReleasesWithFix([]byte) ([]byte, error)
 }
 
+type Conf struct {
+	Flags              []string `yaml:"flags"`
+	IncludeDeployments []string `yaml:"include"`
+}
+
 func NewDeployCmd(
 	ui boshui.UI,
 	deployment boshdir.Deployment,
 	releaseUploader ReleaseUploader,
+	director boshdir.Director,
 ) DeployCmd {
-	return DeployCmd{ui, deployment, releaseUploader}
+	return DeployCmd{ui, deployment, releaseUploader, director}
 }
 
 func (c DeployCmd) Run(opts DeployOpts) error {
 	tpl := boshtpl.NewTemplate(opts.Args.Manifest.Bytes)
 
+	configs, _ := c.director.ListConfigs(1, boshdir.ConfigsFilter{Type: "deploy"})
+
+	for _, config := range configs {
+		var conf Conf
+
+		yaml.Unmarshal([]byte(config.Content), &conf)
+		if conf.IncludeDeployments == nil || includeContains(conf.IncludeDeployments, c.deployment.Name()) {
+			c.ui.PrintLinef("Using deployment flags from config of type '%s' (name: '%s')", config.Type, config.Name)
```

Comment:
Can we make that information more prominent in the cli logs (e.g. color it, or add !!! a line before and afterwards) otherwise it can be overseen to easy.

Additionally maybe the better message would be:
NOTE: The deploy command execution will use parameters that are globally configured in the director's %s config.

Reply:
Sure, we can make it more prominent. I tried to have the text similar to what comes up in the deploy command anyway e.g.

Using deployment 'foo'

So I thought it would be nice to stick to the same sentence "structure"